webpack.config.js

```diff
@@ -12,7 +12,7 @@ const DOT_ENV_CONFIG = getEnv();
 
 // `shell` parameter populated via CLI, e.g. --env.platform=web
 module.exports = (shell = {}) => {
-  const APP_ENV = process.env.APP_ENV || 'live';
+  const APP_ENV = DOT_ENV_CONFIG.parsed.APP_ENV || 'live';
```

Comment:
As dotenv doesn't override existing env variables, this can remain as `process.env.APP_ENV` IMO

Reply:
True, but this feels more explicit in the fact that we don't need to set `process.env.APP_ENV` anymore because it's being set up in the `.env` files. That then means `APP_ENV` will only ever be initialised in a config file and `process.env.ENV_FILE` has a clear purpose of setting up values on entry to the application